core/io/ip.cpp

```diff
@@ -242,12 +301,13 @@ void IP::get_local_addresses(List<IPAddress> *r_addresses) const {
 		}
 	}
 }
-
```

Comment:
Keeping this line of separation would be preferred.

Reply:
Noted